app/views/spotlight/dashboards/_page.html.erb

```diff
@@ -2,8 +2,13 @@
   <td>
     <h4 class="panel-title"><%= page.title %></h4>
     <div class="page-links">
-      <%= exhibit_view_link page, :class => 'btn btn-link' %> &middot;
-      <%= exhibit_edit_link page, :class => 'btn btn-link' %>
+      <% if page.is_a?(Spotlight::HomePage) %>
```

Comment:
I couldn't think of a good way to do this w/o having a lot of extra logic for this one case.  I'm open to suggestions.